e2e/README.md

```diff
@@ -0,0 +1,13 @@
+# E2E Test for this sample app
```

Comment:
still need to set the URL themselves..a Makefile to run pip install + a script seems a bit overkill to me.